bankroll/model/convertablemodel.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import Any, List
+
+
+class ConvertableModel(ABC):
```

Comment:
I'm still having typing issues with this approach.

Reply:
Can you be more specific? What sort of issues?

Reply:
Nit: can we name this `PandasConvertibleModel` (note the `i`) or something like that?

Reply:
✅  Figured out the typing issues